deepchem/feat/base_classes.py

```diff
@@ -150,22 +150,20 @@ def __str__(self) -> str:
     return self.__class__.__name__ + override_args_info
 
 
-class ComplexFeaturizer(object):
+class ComplexFeaturizer(Featurizer):
   """"
   Abstract class for calculating features for mol/protein complexes.
   """
 
-  def featurize(self, mol_files: Sequence[str],
-                protein_pdbs: Sequence[str]) -> Tuple[np.ndarray, List]:
+  def featurize(self, complexes: Iterable[Tuple[str, str]],
+    log_every_n: int = 100) -> np.ndarray:
     """
     Calculate features for mol/protein complexes.
 
     Parameters
     ----------
-    mols: List[str]
-      List of PDB filenames for molecules.
-    protein_pdbs: List[str]
-      List of PDB filenames for proteins.
+    complexes: Iterable[Tuple[str, str]]
+      List of filenames (PDB, SDF, etc.) for ligand molecules and proteins.
```

Comment:
The description is a little ambiguous.  How about adding another sentence: "Each element should be a tuple of the form (ligand_filename, protein_filename)."